### PR TITLE
fix library status reset bug

### PR DESCRIPTION
**Pull request by vevix**
ref: https://forums.hummingbird.me/t/changing-library-info-on-anime-pages-results-in-resets-and-errors/20471

Weird bug, where once we serialize the `LibraryEntry` with its `Anime` object from the server, the relationship between `FullAnime <-> LibraryEntry` gets set to null. ¯\\\_(ツ)_/¯

### CODE REVIEW

File: app/frontend/app/routes/anime/index.js [hunk 1]
```diff
@@ -25,7 +25,12 @@ export default Ember.Route.extend({
       return libraryEntry.save();
     }), {
       progressMessage: "Saving " + anime.get('canonicalTitle') + "...",
-      successMessage: "Saved " + anime.get('canonicalTitle') + "!"
+      successMessage: function() {
```
Comment (NuckChorris) [inline app/frontend/app/routes/anime/index.js]: Does this problem also apply to Manga?

Comment (vevix) [inline app/frontend/app/routes/anime/index.js]: Most likely, I should add some Manga to my db.

**Comment by vevix** [comment]
Applied to Manga as well :+1:

File: app/frontend/app/routes/anime/index.js [hunk 2]
```diff
@@ -25,7 +25,12 @@ export default Ember.Route.extend({
...
+        // update the 'full-anime' relationship, since it seems to
+        // disappear into the void
+        anime.set('libraryEntry', libraryEntry);
+        return "Saved " + anime.get('canonicalTitle') + "!";
+      }
```
Comment (cybrox) [inline app/frontend/app/routes/anime/index.js]: Shouldn't these be `.property()`'s?

Comment (vevix) [inline app/frontend/app/routes/anime/index.js]: nope, they are functions invoked by actions.

Comment (cybrox) [inline app/frontend/app/routes/anime/index.js]: Oh, looked like a top level property in the diff snippet.
Just checked the complete file, you're right.
Looks good to me then.

**Pull request merged**